packages/storybook/src/theme.ts

```diff
@@ -100,11 +100,32 @@ const inputs = {
   },
 }
 
+const switchDefaults = {
+  'input:focus ~ &': {
+    outlineColor: 'blue.1',
+  },
+  '&::before': {
+    background: 'white',
+  },
+}
+
 const forms = {
   select: {
     color: 'black',
     borderColor: 'gray.3',
   },
+  switch: {
+    checked: {
+      backgroundColor: 'blue.1',
+      borderColor: 'blue.4',
+      ...switchDefaults,
+    },
+    unchecked: {
+      backgroundColor: 'grey.5',
+      borderColor: 'grey.5',
+      ...switchDefaults,
+    },
```

Comment:
@alexanderchan this is an interesting application of variants. I'm definitely not against it, it just kind of goes against what was done with the Radio and Checkbox components that managed the Checked/Unchecked SVGs in the component. What would happen if the developer installs Kodiak and they don't add these variants to their theme? They wouldn't have the switchDefaults already set up, would they? That leads me to  think that we need some type of basic defaults for these components on the component level not on the theme level?